src/meta/processors/jobMan/JobManager.cpp

```diff
@@ -79,10 +79,10 @@ void JobManager::shutDown() {
 
 void JobManager::scheduleThread() {
     LOG(INFO) << "JobManager::runJobBackground() enter";
-    while (status_ == Status::RUNNING) {
+    while (status_ != JbmgrStatus::STOPPED) {
```

Comment:
why not while (status_ == JbmgrStatus::IDLE || status_ == JbmgrStatus::BUSY) ？

Reply:
because it's long ...